translations/strings.yaml

```diff
@@ -90,6 +90,15 @@ serversView:
     comment: Label for the “recommended” servers tab
   searchPlaceholder: Search countries
   searchNoResultsLabel: No results. Try a different search
+  recommendedRefreshlLoadingLabel:
+    value: Checking... %1%
```

Comment:
Please check the linter ;-)

Besides that, is it just me or that's a very strange structure? The ellipsis means that the action is in progress, they're never followed by more information

`Checking (%1%)…` would seem more natural to me.

Reply:
> Besides that, is it just me or that's a very strange structure? The ellipsis means that the action is in progress, they're never followed by more information

> Checking (%1%)… would seem more natural to me.

@chrisrooneys - thoughts?

Reply:
I'm good with that change